tests/test_http.py

```diff
@@ -26,6 +27,11 @@ def s():
 
 def has_connection():
     # In case we run tests in an environment without internet connection.
+
+    if sys.version_info < (2, 7):
+        # py 2.6 doesn't have SNI support, required for httpbin, as it has SSLv3 certificates
+        return False
```

Comment:
Its causing SSLError exception in https://travis-ci.org/projectatomic/osbs-client/jobs/235936576:
```
tests/test_http.py:36: in <module>

    @pytest.mark.skipif(not has_connection(),

tests/test_http.py:30: in has_connection

    HttpStream("https://httpbin.org/get", "get")

osbs/http.py:138: in __init__

    self.req = requests.request(method, url, **args)

../../../virtualenv/python2.6.9/lib/python2.6/site-packages/requests/api.py:58: in request

    return session.request(method=method, url=url, **kwargs)

../../../virtualenv/python2.6.9/lib/python2.6/site-packages/requests/sessions.py:518: in request

    resp = self.send(prep, **send_kwargs)

../../../virtualenv/python2.6.9/lib/python2.6/site-packages/requests/sessions.py:639: in send

    r = adapter.send(request, **kwargs)

../../../virtualenv/python2.6.9/lib/python2.6/site-packages/requests/adapters.py:512: in send

    raise SSLError(e, request=request)

E   SSLError: [Errno 1] _ssl.c:493: error:14077438:SSL routines:SSL23_GET_SERVER_HELLO:tlsv1 alert internal error
```